modules/ivc_champva/spec/jobs/missing_form_status_job_spec.rb

```diff
@@ -18,35 +18,11 @@
     expect(StatsD).to have_received(:gauge).with('ivc_champva.forms_missing_status.count', forms.count)
   end
 
-  it 'sends each form UUID to DataDog' do
-    IvcChampva::MissingFormStatusJob.new.perform
-
-    forms.each do |form|
-      expect(StatsD).to have_received(:increment).with('ivc_champva.form_missing_status', tags: ["id:#{form.id}"])
-    end
-  end
-
   it 'logs an error if an exception occurs' do
     allow(IvcChampvaForm).to receive(:where).and_raise(StandardError.new('Something went wrong'))
 
     expect(Rails.logger).to receive(:error).twice
 
     IvcChampva::MissingFormStatusJob.new.perform
   end
-
-  context 'Feature champva_failure_email_job_enabled=true' do
-    before do
-      allow(Flipper).to receive(:enabled?)
-        .with(:champva_failure_email_job_enabled, @current_user)
-        .and_return(true)
-    end
-
-    it 'sends count of forms missing Pega status for >= 7 days to DataDog' do
-      IvcChampva::MissingFormStatusJob.new.perform
-      forms.each do |form|
-        expect(StatsD).to have_received(:increment)
-          .with('ivc_champva.form_missing_status_email_sent', tags: ["id:#{form.id}"])
-      end
-    end
-  end
 end
```

Comment:
These tests don't work now that we're using the monitor method. Added a test to the monitor spec file below to account for this change.